pkg/service.go

```diff
@@ -372,6 +372,8 @@ func (s *DefaultService) GetPayloadDelivered(ctx context.Context, query TraceQue
 
 	if err == nil {
 		return []types.BidTrace{event.BidTrace}, err
+	} else if errors.Is(err, ds.ErrNotFound) {
+		return []types.BidTrace{}, nil
```

Comment:
Can we return `nil` instead of allocating a `[]types.BidTrace{}`?

Reply:
Not really, since we must return an empty slice to comply with other relays